documentation/en/api-v0-methods-miner.md

```diff
@@ -1087,7 +1087,7 @@ Response:
         }
       },
       "Diagnostics": [
-        "string value"
```

Comment:
We'll need to introduce the Ethereum address type even if it's just an alias for now, to avoid example mangling.

Reply:
introduced a EthAddress type